page_analyzer/utils/db.py

```diff
@@ -0,0 +1,15 @@
+import os
+from typing import Any
+
+import psycopg2
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+DATABASE_URL = os.getenv("DATABASE_URL")
+
+
```

Comment:
[just an idea] move code for getting URL data and URL creation here as well (to encapsulate all db-related logic in this module)